devito/ir/iet/analysis.py

```diff
@@ -122,16 +128,15 @@ def mark_parallel(analysis):
 
 
 @propertizer
-def mark_vectorizable(analysis):
-    """Update the ``analysis`` detecting the ``VECTOR`` Iterations within
-    ``analysis.iet``. An Iteration is VECTOR iff:
-
-        * it's the innermost in an Iteration tree, AND
-        * it's got at least an outer PARALLEL Iteration, AND
-        * it's been marked as PARALLEL or all the accesses along its dimension
-          are unit-strided.
-        """
+def mark_iteration_vectorizable(analysis):
+    """
+    Update the ``analysis`` detecting the VECTOR Iterations within ``analysis.iet``.
+    """
     for tree in analysis.trees:
+        # An Iteration is VECTOR iff:
+        # * it's the innermost in an Iteration tree, AND
+        # * it's got at least an outer PARALLEL Iteration, AND
```

Comment:
Is this really true?

Reply:
wdym ?

surely a fully parallel loop is also a vectorizable one. Above is just an heuristic to avoid marking all loops in a nest as VECTOR so that the DLE will only find (at most) one per nest

Reply:
ah! I think it's sufficient, not necessary. 
However, the `iff` refers to marking the Iteration as a VECTOR one, not to whether it's theoretically vectorizable or not (hence the property name VECTOR, as opposed to VECTORIZABLE)